### ISSUE TITLE
bug: fig window position in RubyMine terminal

### ISSUE THREAD

**Issue by BenoitDeo** [withfig/fig#2146]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 13.0.1 (22A400)

### Expected behaviour

expect the position of the fig window to be at cursor level in RubyMine terminal

### Actual behaviour

position of the fig window is several cm bellow the cursor level in RubyMine terminal

### Environment

```yaml
fig-details:
  - 2.9.0
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1 Pro
  - cores: 8
  - mem: 16.00 GB
os-info:
  - macOS 13.0.1 (22A400)
environment:
  - shell: /Applications/Fig.app/Contents/MacOS/fig_desktop
  - terminal: <unknown>
  - cwd: /
  - exe-path: /Applications/Fig.app/Contents/MacOS/fig-darwin-universal
  - install-method: unknown
  - env-vars:
    - __CFBundleIdentifier: com.mschrage.fig
    - PATH: /usr/bin:/bin:/usr/sbin:/sbin
    - SHELL: /bin/zsh
```

**Comment by grant0417** [comment]
This should be resolved now in Fig 2.10, let me know if you continue to have any issues!

**Comment by BenoitDeo** [comment]
Awesome, thanks!

**Comment by BenoitDeo** [comment]
I restarted and updated fig, and I can see the fig window offset several cm to top and right of the cursor.

version: RubyMine 2022.2.3
Please let me know if there any other relevant information that you may need.

**Comment by grant0417** [comment]
Hey @BenoitDeo, sorry for the long delay on this but we just released 2.11 which should hopefully resolve these issues!

Let me know if this fixes the issue

**Comment by BenoitDeo** [comment]
hey Grant, no worries. I upgraded and restarted my IDE and seems like that the issue is now solved! Thanks for looking and fixing this issue, much appreciated!
